web/index.html

```diff
@@ -21,10 +21,6 @@
     </head>
     <body class='h-full'>
         <div id="app" class="h-full relative">
-            <div class='w240 align-center'>
-                LOGIN
-
-            </div>
```

Comment:
Can I get some context on this deletion? Is it just dead code?

Reply:
Yup, this was code I added when I was testing out the full auth mode that we currently have deployed. VueJS takes control of the parent div and wipes anything inside of it (name this). So you would briefly see `LOGIN` flash by on the screen if you're browser was a mite slow.